docs/src/examples/Lazy-abstraction-reachability.jl

```diff
@@ -21,25 +21,28 @@
 # at the same time as the controller.
 
 # First, let us import [StaticArrays](https://github.com/JuliaArrays/StaticArrays.jl) and [Plots].
-using StaticArrays, Plots
+using StaticArrays, JuMP, Plots
 
 # At this point, we import Dionysos.
 using Dionysos
 const DI = Dionysos
 const UT = DI.Utils
 const DO = DI.Domain
 const ST = DI.System
-const CO = DI.Control
 const SY = DI.Symbolic
-const LA = DI.Control.LazyAbstractionReach
+const CO = DI.Control
+const PR = DI.Problem
+const OP = DI.Optim
+const AB = OP.Abstraction
+
+include("../../../problems/SimpleProblem.jl")
```

Comment:
We should be careful about how we call files
I think we should uniformize like this_is_my_file.jl, and ThisIsMyModule